### ISSUE TITLE
Níveis de dificuldade?

### ISSUE THREAD

**Issue by azevgabriel** [azevgabriel/educacao_especial#10]
**Modo fácil** é da maneira que estamos implementando hoje!

**Modo médio:** seria sem a primeira palavra de apoio.
![image](https://user-images.githubusercontent.com/73303855/172016484-fcd21a74-dc48-45ff-8aa1-cce1130bd3b0.png)

**Modo difícil:** sem a palavra de apoio e com apenas uma silhueta do animal.
![image](https://user-images.githubusercontent.com/73303855/172016632-075adfe7-91e1-41c0-8f85-436bd279e09c.png)

**Comment by azevgabriel** [comment]
Outra opção é darmos apenas algumas letras. Exemplo: B-?-?-E

**Comment by leoalmelin** [comment]
Gostei das suas sugestões para os modos de dificuldade. Retirar as letras acho que seria muito mais difícil, talvez possamos pensar em outros exercícios com focos para a escrita sem apoio. Poderia ser algo voltado em uma etapa mais avançada no aprendizado das crianças.

Sua ideia é implementarmos os três níveis de dificuldade para todos os jogos?

**Comment by azevgabriel** [comment]
Minha ideia é criar uma progressão de acordo com o avanço do aluno. Mas ai temos que tomar um cuidado com os alunos que não retém o conteúdo. Por isso á ideia de colocar algo que verifica a taxa de acerto do aluno, a gente dificulta o jogo sem o aluno ter que necessariamente escolher um nível. 

Sobre a quantidade de dificuldades, não seria necessariamente 3. Mas quantas a gente conseguir pensar, sem ter um degrau de complexidade muito alta de nível para nível.

**Comment by leoalmelin** [comment]
Ótima ideia. 

Para essa passagem autómatica de nível seria necessário o login indivual de cada aluno, correto? Acredito que isso seria então uma consequência de quando e como isso será implementado.

Podemos colocar essa troca de dificuldade de forma manual neste primeiro momento? Ou acha melhor seguir com isso depois, juntamento com a análise das taxas de acerto?

**Comment by azevgabriel** [comment]
@leoalmelin , perfeito. Acho que o caminho, é fazemos de forma manual e depois implementamos essa lógica. Até para ter algo pronto pra uso, mesmo que nossas ideias não tenham acabado.

> Resposta à questão secundária.

Já vi algumas coisa legais. Como barra de progresso que passa de nível, ou alguma medalhas de condecorações. Sobre você equipar seu perfil com essas medalhas para mostrar para os outros colegas que você conseguiu aquela medalha. Porém eu acho que a gente entra também em um lado de competitividade, quando da um progresso pra um e pra outro colega não, como a gente equilibra isso de uma forma natural?

**Comment by leoalmelin** [comment]
> Resposta à questão secundária.

@azevgabriel Uma alternativa seria que funcionassem como os carimbos das atividades usuais. O educador poderia ter autonomia para distribuir essas medalhas (poderíamos alterar as cores dos modelos para aumentar a variação de cada uma), e distribuir de acordo com a finalização das atividades diárias. Assim, poderiam ter medalhas de acordo com objetivos específicos, e essas gerais, e o educador guiar a distribuição das gerais de forma a equilibrar da melhor forma. Isso deve ser um dos pontos a levantarmos e discutirmos nas prováveis conversas que teremos com os educadores no futuro.

**Comment by azevgabriel** [comment]
@leoalmelin Eu pensei em medalhas de esforço ao invés de mérito. Como por exemplo, medalha Fazendeiro, completou mais de 20 atividades de tema de fazenda. Pois independente da pontuação ou do tempo, estamos avaliando se eles são persistentes em tentar até o fim.

> Uma alternativa seria que funcionassem como os carimbos das atividades usuais. O educador poderia ter autonomia para distribuir essas medalhas (poderíamos alterar as cores dos modelos para aumentar a variação de cada uma), e distribuir de acordo com a finalização das atividades diárias. Assim, poderiam ter medalhas de acordo com objetivos específicos, e essas gerais, e o educador guiar a distribuição das gerais de forma a equilibrar da melhor forma. Isso deve ser um dos pontos a levantarmos e discutirmos nas prováveis conversas que teremos com os educadores no futuro.

Como disponibilizaríamos isso pro educador. Ele teria acesso à todos os designs de medalhas para serem impressos, ou ele teria seu login de educador e daria as medalhas de acordo com o seu saber?

**Comment by leoalmelin** [comment]
@azevgabriel sim, concordo com as medalhas por esforço. Mas ainda assim poderia ter um desequilíbrio de acordo com o tempo que cada um leva para terminar as atividades.

> Como disponibilizaríamos isso pro educador. Ele teria acesso à todos os designs de medalhas para serem impressos, ou ele teria seu login de educador e daria as medalhas de acordo com o seu saber?

Acho que o interessante seria o login de educador, assim ele também teria acesso aos dados individuais dos seus alunos. Pensei em colocarmos um menu que pudessem selecionar uma medalha e para qual aluno da sua turma atribuir. E fazermos algumas medalhes específicas para isso, e ele poderia selecionar entre estas. Como se fossem os carimbos mesmo. Isso seria uma etapa a ser feita bem mais a frente no projeto.
